src/Chapter1.hs

```diff
@@ -641,7 +662,11 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Integral a => a -> a
+firstDigit n
+  | n >= 10 = firstDigit (n `div` 10)
+  | n < 0 = firstDigit(- n)
```

Comment:
Nice solution 👍🏼 

It is better to separate function and arguments with spaces, even when it is in parenthesis 🙂 
```suggestion
  | n < 0 = firstDigit (- n)
```